src/App.js

```diff
@@ -0,0 +1,63 @@
+import './App.css';
+import React from 'react';
+import Dynamic from './Dynamic';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.setBackgroundImage();
+  }
+
+  setBackgroundImage() {
+    const backgroundImage = "#base64BackgroundImage"
+    if (!backgroundImage.includes('#'))
+      base64DynamoBackground = backgroundImage;
+  }
+
+  render() {
+    return (
+      <Container className='fill' fluid="md">
+        <Row>
+          <Col className='menuOptions' >
+            <Row className='bottomMenu'>
+              <Col>
+                <Row>
+                  <div>
+                    <img className='dynamoLogo' src={base64DynamoLogo}></img>
+                  </div>
+                </Row>
+                <Row className='welcomeRow'>
+                  <div >
+                    Welcome to Dynamo!
```

Comment:
Can you make this settable from Dynamo but with this default value? I ask only because we currently have a blocker on the localization side, so we may need to use Dynamo to pass in localized string for this one. The other resources are fine, e.g. the title `Dynamo` does not need to be localized